src/ui/components/atoms/tag/Tag.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from './styles.module.scss';
+import type { IProps } from './types/IProps';
+
+function Tag({ children, variant = 'primary', onClick }: IProps) {
+  const variantClass = styles[`tag--${variant}`];
```

Comment:
In order to maintain a good code here please use the approach set up in the demo component. 
example here 
https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx#L7